clang-tools-extra/test/clang-doc/clang-doc-project1.cpp

```diff
@@ -0,0 +1,370 @@
+// RUN: mkdir -p %T/clang-doc/build
+// RUN: mkdir -p %T/clang-doc/include
+// RUN: mkdir -p %T/clang-doc/src
+// RUN: mkdir -p %T/clang-doc/docs
```

Comment:
```suggestion
// RUN: mkdir -p %t/clang-doc/build
// RUN: mkdir -p %t/clang-doc/include
// RUN: mkdir -p %t/clang-doc/src
// RUN: mkdir -p %t/clang-doc/docs
```
`%T` is deprecated. https://llvm.org/docs/TestingGuide.html#substitutions
Also,  begin w/ `rm -rf %t`. It's probaly also fine to  rewrite the whole thing as a one-liner.
```suggestion
// RUN: rm -rf %t && mkdir -p %t/clang-doc/build %t/clang-doc/include %t/clang-doc/src %t/clang-doc/docs
```